packages/graalvm/src/test/kotlin/elide/runtime/gvm/internals/js/node/NodeFsPromisesTest.kt

```diff
@@ -21,10 +21,11 @@ import elide.testing.annotations.TestCase
 
 /** Tests for Elide's implementation of the Node `fs/promises` built-in module. */
 @TestCase internal class NodeFsPromisesTest : NodeModuleConformanceTest<NodeFilesystemModule>() {
-  override val moduleName: String get() = "fs/promises"
-  override fun provide(): NodeFilesystemModule = NodeFilesystemModule()
   @Inject lateinit var filesystem: NodeFilesystemModule
 
+  override val moduleName: String get() = "fs/promises"
+  override fun provide(): NodeFilesystemModule = filesystem
```

Comment:
The value used in the tests must be properly injected by Micronaut, otherwise injected fields will not be available.